test/Reflection/ReflectionFunctionTest.php

```diff
@@ -193,12 +195,14 @@ function function2(): void
 {
 }
 
+// phpcs:disable
 /**
  * \LaminasTest\Server\Reflection\function3
  *
  * @param  string $var1
  * @return void
  */
+// phpcs:enable
 function function3(string $var1): void
```

Comment:
These are largely a remnant from the days when visibility wasn't used often. I think it might make more sense to update these, add visibility, and update the docblocks, instead of ignoring phpcs.

Reply:
CS is ignored here, since DocBlock is required for reflection of the component. CS will fail validation due to redundant param and return tags. Removing DocBlock will fail test `ReflectionFunctionTest::testMultipleWhitespaceBetweenDoctagsAndTypes`.